cmd/wallet-sdk-gomobile/scripts/build_android_bindings.sh

```diff
@@ -0,0 +1,7 @@
+# This script needs to be called from the same folder that the wallet-sdk-gomobile makefile is in.
```

Comment:
Can we add licence header to these new files ?

Reply:
Done!